docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -55,11 +55,11 @@ Ensure that the following prerequisites are met:
 - Users who require access to mainframe resources using OIDC authentication have a mainframe identity managed by SAF/ESM.
 - Client application users have their distributed identity managed by the OIDC provider. For details, see the section [OIDC provider](#oidc-provider) in this topic.
 - SAF/ESM is configured with mapping between the mainframe and distributed user identities. For details, see the section [ESM configuration](#esm-configuration) in this topic.
-- ZSS must be enabled and properly configured in the Zowe installation.
+- Ensure that the API ML Gateway is configured to use the internal mapper functionality. Alternatively, enable ZSS in the Zowe installation, however using the internal mapper is the recommended method. ZSS is enabled by default.
```

Comment:
I think this is missing either a link to the internal mapper doc or at least the minimum version of Zowe?

Reply:
Thank you